pysiaf/siaf.py

```diff
@@ -327,8 +327,8 @@ def apernames(self):
         return self.apertures.keys()
 
     def plot(self, frame='tel', names=None, label=None, units=None, clear=True, annotate=False,
-             mark_ref=False, subarrays=True, **kwargs):
-        """Plot all apertures in this SIAF.
+             origin="both", mark_ref=False, subarrays=True, ax=None, **kwargs):
+        """ Plot all apertures in this SIAF.
```

Comment:
PEP257 requires that there is no space after the triple quotes: https://www.python.org/dev/peps/pep-0257/#specification